tests_integration/test_connector_flows.py

```diff
@@ -116,18 +99,17 @@ def test_connector_flows(run_kopf, random_connector_name):
                 }
             },
         )
-        time.sleep(10)
-        pod = kubectl_get("pod", connector_name)
+        time.sleep(5)
```

Comment:
I think it's nicer to delete this and let the retry logic in `kubectl_wait_pod_running` to sleep if neccessary.